docs/references/cluster-metrics.mdx

```diff
@@ -263,3 +263,79 @@ Number of Workflows that successfully completed.
 ### `workflow_timeout`
 
 Number of Workflows that timed out before completing execution.
+
+## Nexus metrics
+
+These metrics pertain to Nexus Operations.
+
+### Nexus Machinery in the History Service
+
+See [architecture
+document](https://github.com/temporalio/temporal/blob/5d55d6c707bd68d8f3274c57ae702331adf05e6e/docs/architecture/nexus.md#scheduler)
+for more info.
+
+#### In-Memory Buffer
+
+- `dynamic_worker_pool_scheduler_enqueued_tasks` - a counter that is incremented when a task is enqueued to the buffer
+- `dynamic_worker_pool_scheduler_dequeued_tasks` - a counter that is incremented when a task is dequeued from the buffer
+- `dynamic_worker_pool_scheduler_rejected_tasks` - a counter that is incremented when the buffer is full and adding the
+  task is rejected
+- `dynamic_worker_pool_scheduler_buffer_size` - a gauge that periodically samples the size of the buffer
+
+### Concurrency Limiter
+
+- `dynamic_worker_pool_scheduler_active_workers` - a gauge that periodically samples the number of running goroutines
+
+#### Rate Limiter
+
+- `rate_limited_task_runnable_wait_time` - a histogram representing the time a task spends waiting for the rate limiter
+
+#### Circuit Breaker
+
+- `circuit_breaker_executable_blocked` - a counter that is incremented every time a task execution is blocked by the
+  circuit breaker
+
+#### Task Executors
+
+- `nexus_outbound_requests` - a counter representing the number of Nexus outbound requests made by the history service
```

Comment:
Looks great and merge ready! I'd just change the dashes to full sentences with a colon. 

```suggestion 
`nexus_outbound_requests`: A counter representing the number of Nexus outbound requests made by the history service.
```